package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.12.2",
     "serve-static": "1.9.2",
     "shelljs": "0.4.0",
+    "sinon": ">=1.4.0 <2",
```

Comment:
same as comment on other PR https://github.com/tidepool-org/message-api/pull/34 this is a departure from our current way of requiring versions. Open to ideas  FYI @jebeck